frontend/src/app/views/boards/opponent-board/opponent-board.component.css

```diff
@@ -0,0 +1,28 @@
+.field {
+  display: inline-block;
+  font-size: 0;
+  margin: 10px 0;
```

Comment:
consider adding flexible margin e.g.  10%, one css technics is a flexbox

Reply:
done